### PR TITLE
Vmo 4395 fixes for toolbar numeric block

### PR DESCRIPTION
**Pull request by safydy-r**
I took the opportunity to upgrade MinimumNumericEditor & MaximumNumericEditor into class-based components.

Overview
vmo4395

### CODE REVIEW

File: src/components/common/NumericEditor.vue [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default {
     },
     value: {
       type: [String, Number],
-      required: true,
+      required: false,
```
Comment (safydy-r) [inline src/components/common/NumericEditor.vue:40]: this allows us to provide `undefined` value

File: src/components/interaction-designer/block-editors/MaximumNumericEditor.vue [hunk 2]
```diff
@@ -13,40 +13,30 @@
   </validation-message>
 </template>
 
-<script lang="js">
+<script lang="ts">
```
Comment (safydy-r) [inline src/components/interaction-designer/block-editors/MaximumNumericEditor.vue:16]: Took the opportunity to upgrade to class-based component

File: builder.config.json [hunk 3]
```diff
@@ -191,16 +191,16 @@
         "is_interactive": true,
         "is_branching": false,
         "category":  1,
-        "menu_category": 4
+        "menu_category": 1,
```
Comment (poojahpatil90) [inline builder.config.json:194]: Curiosity Q: Are we using the `menu_category` to sort the block types between different categories in the new toolbar?

**Review by poojahpatil90 — APPROVED** [review]
Everything looks good to me. Approved!

File: builder.config.json (hunk 3 above)
Comment (safydy-r) [reply builder.config.json:194]: I think no, this is just for menu.

**Pull request merged**